ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -168,16 +168,16 @@ public struct AvatarGroup: View {
                 // If the avatar is part of Stack style and is not the last avatar in the sequence, create a cutout
                 let avatar = avatars[index]
                 let avatarView = Avatar(avatar)
+                let needsCutout = tokens.style == .stack && (overflowCount > 0 || index + 1 < maxDisplayedAvatars)
                 let avatarSize = avatarView.size()
-                let nextAvatarSize = Avatar(avatars[index + 1]).size()
+                let nextAvatarSize: CGFloat = needsCutout ? Avatar(avatars[index + 1]).size() : 0
```

Comment:
This bit is failing the code again, similar to the arithmetic issues we ran into in #648... Any ideas on this?